credits.txt

```diff
@@ -16,9 +16,16 @@ Developers
   Amazinite
   Benjamin Hauch
   MCOfficer
+  Peter van der Meer
   Pointedstick
+  Quyykk
   W1zrad
 
+Project Administration
+  Michael Zahniser
+  Benjamin Hauch
+  Zitchas
+
```

Comment:
Don't really see the point in this credit.

Reply:
Document who is responsible for administration? No sense keeping it a secret.

Reply:
Just because it isn't in the game credits doesn't mean it's a secret.

Reply:
I don't see why it's necessary though. If people aren't exposed to github, they have no tangible benefit of knowing who runs the show.

The true purpose of the credits is to give something back, to give contributors a place in the spotlight. That's the angle from which this should be approached: is management important enough to deserve an extra section? Or are you lot content with your current positions in the credits?

Reply:
Two thoughts:
a) I thought about just adding my name on the end of the developers section, but that's not particularly accurate. General consensus is that I'm not a Dev. (or perhaps more accurately, I'm the Dev of last resort if everyone else in the list falls off the face of the planet. At which point I'll be more looking for new devs rather than reviewing). 

b) If we just want to trim that down to 
> Project Administration
>    Zitchas
I'm fine with that. The other two are already in the credits in what I'd term to be more important roles anyway. I would like to be on there, though.

Reply:
Is being under the storyline and faction author section not sufficient enough?

Reply:
> Is being under the storyline and faction author section not sufficient enough?

No, it feels to me like it's not enough to reflect what Zitchas has done to make ES a success; If we list Developers separately, then I feel it is fine and very appropriate to also list Zitchas as Developer. Not all development is about code and artwork; Zitchas would be listed in a "community building" type of developer role.

I do agree with other Developers that listing "Project Administrators" look a bit strange in the credits; people deeply involved in the project already know who does what, and people that know nothing about the project background would likely not understand what a "Project Administrator" does.

Reply:
I listed myself under an administrative role, as basically my involvement on GitHub is in that capacity. I'm one of three people who can manage who has write permissions, appoint Devs, as well as a variety of other things. So in that sense, I'm not just another Dev. But you have a good point that this is basically just an internal distinction, so I've changed it so now I'm just in the Dev list. I'm fine with that.